build/builder.go

```diff
@@ -327,6 +327,15 @@ func (o *Options) IndexState() IndexState {
 		return IndexStateCorrupt
 	}
 
+	ts, err := zoekt.LoadTombstones(fn)
```

Comment:
instead of handling here, what do you think about handling this in `findShard`? findShard needs to be tombstone aware due to compound shards.